tests/features/splash_screen.feature

```diff
@@ -50,3 +49,6 @@ Feature: Splash Screen features
     And I should see the link "Deutsch"
     And I should not see "Български"
     And I should not see "Français"
+    # Clean up after the fact.
+    # @todo Remove this once NEXTEUROPA-5519 is in.
+    And I run drush "vdel" "splash_screen_blacklist --yes"
```

Comment:
Can be deleted (not necessary to wait the fix of NEXTEUROPA-5519)
Currently, enabled module(s) are uninstalled at the end of scenario   (see ``cleanModule()``), and the variable is deleted on the ``splash_screen_uninstall`` hook.
(I did not realize that the background is executed for each scenario (so the desinstall of modules in the @afterscenario is executed for each scenario)